package.json

```diff
@@ -79,12 +79,11 @@
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^2.0.1",
     "karma-webpack": "^2.0.5",
-    "lerna": "^3.10.5",
+    "lerna": "^4.0.0",
```

Comment:
This wasn't strictly speaking necessary, but I was chasing down rabbits and...well, it's now at the latest version and everything still works as expected as far as I can tell.

Reply:
Yeah, it doesn't look like any of the [breaking changes](https://github.com/lerna/lerna/blob/main/CHANGELOG.md#breaking-changes) should affect anything.